src/main/scala/esmeta/analyzer/AbsSemantics.scala

```diff
@@ -161,6 +161,41 @@ class AbsSemantics(
   def errors: Set[TypeError] =
     errorMap.map((_, innerMap) => innerMap.values.reduce(_ + _)).toSet
 
+  def unreachables: Set[Unreachable] = {
+    val funcs = (for {
+      func <- cfg.funcs
+      entry = func.entry
+    } yield func -> entry).toSet
+    var set = Set[Unreachable]()
+    val visited = MMap[Node, Boolean]()
+    val total = npMap.keys.map(_.node).toSet
```

Comment:
It seems like this variable(`total`) is used nowhere.

Reply:
thanks :) I fixed it.